sqlmesh/utils/__init__.py

```diff
@@ -63,6 +64,13 @@ def __setitem__(self, k: KEY, v: VALUE) -> None:
 class AttributeDict(dict, t.Mapping[KEY, VALUE]):
     __getattr__ = dict.get
 
+    def __deepcopy__(self, memo: t.Dict[t.Any, AttributeDict]) -> AttributeDict:
```

Comment:
what's this doing?

Reply:
The deepcopy for this class is broken otherwise.